packages/zone/src/durable.js

```diff
@@ -89,6 +89,10 @@ export const makeDurableZone = (baggage, baseLabel = 'durableZone') => {
   /** @type {import('.').Zone['exoClass']} */
   const exoClass = (...args) => prepareExoClass(baggage, ...args);
   /** @type {import('.').Zone['exoClassKit']} */
+  // @ts-ignore This type check regressed inexplicably with the release
```

Comment:
I'll take a look at fixing this before approving

Reply:
Likewise, although it would be nice to get this resolved now, it can happen after this PR, so I won't withhold approval waiting for it.